websites/H/Holodex/presence.ts

```diff
@@ -169,26 +175,12 @@ const getInfo = {
         default:
           return {
             image: "largeimage",
-            hover: ""
+            hover: undefined
```

Comment:
No undefined, already told you :/

Reply:
Oops, sorry I was going to add some edge-case string there but forgot to, I already covered all of the paths the website can have so I'll just set it to "Holodex" or something if it breaks for some reason.
Btw, rn the state is also a blank sting as the default but there isn't really anything meaningful I can put there in pages like about, settings or login. Do I **have to** change that too?

Reply:
If blank string will not be send to discord, that's fine, but if it will be, that's not good.

Reply:
So i can set the state like i set the buttons?
```js
if(data.state.length > 0)
    presenceData.state = data.state;
```

Reply:
```javascript
if(data.state)
    presenceData.state = data.state;
```
why you want check length in string xD